docs/manual/build_system/prerequisites.qbk

```diff
@@ -193,6 +192,29 @@ detail in the sections __unix_installation__ and __windows_installation__.
 
 [endsect]
 
+[/////////////////////////////////////////////////////////////////////////////]
+[section:hwloc_installation Installing Hwloc]
+
+We suggest you download the most recent release of hwloc from here:
+__hwloc_downloads__. Unpack the downloaded archive into a directory of your
+choosing. We will refer to this directory as `$HWLOC`.
+
+To build hwloc run:
+
+    cd $HWLOC
+    ./configure --prefix=<where to install hwloc>
+    make -j<N> install
+
+where: `<where to install hwloc>` is the directory the built binaries will be
+installed to, and `<N>` is the number of cores to use to build hwloc.
+
+After the above sequence of commands has been executed you will need to specify
+the directory where Hwloc was installed as `HWLOC_ROOT` (`<where to install
+hwloc>`) while executing cmake for __hpx__ as explained in detail in the
+sections __unix_installation__ and __windows_installation__.
+
+[endsect]
```

Comment:
Could you please add a note that the instructions don't refer to Windows based installations. I'm not sure about MacOS and/or Android based systems (@sithhell?). For Windows the user can simply download the 64Bit binaries from the download page.

Reply:
I've added a note. Could you check that it's accurate (I wrote a one-sentence explanation for how to install hwloc on Windows)?

Reply:
I suggest to link to the hwloc documentation for further information:
https://www.open-mpi.org/projects/hwloc/doc/v1.11.9/
(There's no direct link to the installation section)

From a cursory look, it seems that getting hwloc to run on android is possible (but there's no real documentation), for macosx, I guess that autotools just works? Or homebrew or something similar.

Reply:
@sithhell I've added the hwloc link as a note at the end of the installation section.